Libraries/Text/Text/RCTFontScaleRamp.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import <React/RCTConvert.h>
+
+typedef NS_ENUM(NSInteger, RCTFontScaleRamp) {
+  RCTFontScaleRampUndefined,
```

Comment:
Do we have a need for an `Undefined` value if we want to default to `Body`?

Reply:
Ah I see, it's used above for a fallback value if this property is undefined.

Reply:
Also, `Undefined` means that we use the default linear scaling mechanism that's already baked into RN, while `Body` specifically uses the body font ramp.